src/nic.c

```diff
@@ -38,15 +38,29 @@
 #define TCP_PROTOCOL_NUMBER 6
 #define UDP_PROTOCOL_NUMBER 17
 
-static inline int pg_rte_devargs_remove(struct rte_devargs *devargs)
+static inline int pg_set_nic_port(char *ifname)
 {
+	uint16_t portid;
 
 #if (RTE_VERSION_NUM(18, 5, 0, 0) > RTE_VERSION)
 #error"DPDK is too old"
 #elif (RTE_VERSION_NUM(18, 11, 0, 0) > RTE_VERSION)
```

Comment:
18.02 should work with rte_eth_dev_attach
also, you might need to readd deprecated to gcc